zipline/finance/performance/position_tracker.py

```diff
@@ -258,6 +258,19 @@ def execute_transaction(self, txn):
             position = self.positions[sid]
 
         position.update(txn)
+
+        if position.amount == 0:
+            # if this position now has 0 shares, remove it from our internal
+            # bookkeeping.
+            del self.positions[sid]
+
+            try:
+                # if this position exists in our user-facing dictionary,
+                # remove it as well.  
+                del self._positions_store[sid]
```

Comment:
Are these the cases where it wouldn't exist?

1) `context.positions` has not been called so positions_store has not been populated.
2) user code running `del self._positions_store[sid]`